cfme/tests/infrastructure/test_provisioning_rest.py

```diff
@@ -110,6 +110,40 @@ def test_provision(request, appliance, provision_data):
     assert found_vms, 'VM `{}` not found'.format(vm_name)
 
 
+@pytest.mark.meta(server_roles="+notifier")
+def test_provision_emails(request, provision_data, provider, appliance, smtp_test):
+    """
+    Test that redundant e-mails are not received when provisioning VM that has some
+    attributes set to values that differ from template's default.
+
+    Metadata:
+        test_flag: rest, provision
+    """
+    def check_one_approval_mail_received():
+        return len(smtp_test.get_emails(
+            subject_like="%%Your Virtual Machine configuration was Approved%%")) == 1
+
+    def check_one_completed_mail_received():
+        return len(smtp_test.get_emails(
+            subject_like="%%Your virtual machine request has Completed%%")) == 1
+
+    request.addfinalizer(lambda: clean_vm(appliance, vm_name))
+
+    vm_name = provision_data["vm_fields"]["vm_name"]
+    provision_data["vm_fields"]["vm_memory"] = "1024"
```

Comment:
Since this is statically assigned, it could actually be same as the existing value, couldn't it?

I think it would be better to instead modify the existing value in `provisioning_data` by adding a small amount to `vm_memory` and adding `1` to `number_of_sockets`